jmix-flowui/flowui/src/test/resources/main_view/view/main-view.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<mainView xmlns="http://jmix.io/schema/flowui/main-view"
```

Comment:
Main view already exists in tests: `TestMainView`. May be we can move testing code in it?

Reply:
I saw this but decided against it so as not to affect other views.